recipe/patches/0003-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -1,18 +1,8 @@
-From 5ba4c3b61656a28326469356f286d64e29a1e957 Mon Sep 17 00:00:00 2001
-From: Kai Fricke <kai@anyscale.com>
-Date: Fri, 3 Dec 2021 10:55:23 -0800
-Subject: [PATCH 4/6] Remove all dependencies from setup.py
-
-Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
----
- python/setup.py | 61 ++++++++-----------------------------------------
- 1 file changed, 9 insertions(+), 52 deletions(-)
```

Comment:
is there a reason to strip headers?

Reply:
The patch changed significantly, and the information in the header is no longer correct. I wouldn't want the blame for mistakes to be attributed to the original author.

Reply:
I restored the removed lines here with a disclaimer that I made subsequent changes.